text/0000-discord.md

```diff
@@ -0,0 +1,170 @@
+
+- notifications
+- muting a channel
+- muting a category
+
+Because our goal is transparency, all channels will be visible. A lock icon will display if the user does not have the role necessary to join that channel.
+
+The following proposed initial category and channel list was chosen based on the current channel needs and evaluation of the channels with the most members on Slack. _Additional channels may be requested in the Admin/community-feedback channel._
+
+**Category/Channel List:**
+- (No Category)
+  - welcome (community guidelines are posted here)
+  - setup-profile (how to setup your profile)
+- Admin
+  - community-feedback (questions, comments, concerns, requests)
+  - security
+  - steering-committee 🔒 (locked to role “steering-committee”)
+  - news & announcements
+  - ember-jobs
+  - bots
+- Core Teams
+  - ember-js 🔒 (locked to role “core-js”)
+  - ember-data 🔒 (locked to role “core-data”)
+  - ember-cli 🔒 (locked to role “core-cli”)
+  - ember-learning 🔒 (locked to role “core-learning”)
+- Working on Ember
+  - ember-cli
+  - ember-data
+  - ember-engines
+  - ember-js
+  - glimmer-vm
+  - triage
+  - st-* (as needed)
+- Using Ember
+  - general-help
+  - learning-ember
+  - a11y
+  - backend
+  - internationalization
+  - jsonapi
+  - mobile
+  - ember-js
+  - ember-data
+  - ember-cli
+  - ember-engines
+  - fastboot
+  - ember-twiddle
+  - e-*
+- Supporting Ember
+  - documentation
+  - website
+  - marketing-and-advocacy
+  - infrastructure
+- Event-Chat
+  - EmberConf
+  - EmberCamp
+  - EmberFest
+  - Talks
+  - Other Conferences
+  - Meetup organizers
+- Social
+  - Water-cooler (random)
+  - Local-*
+  - Media (livestreams, videos, podcasts)
+  - Pets
+  - Women in Ember 🔒
+
+## How do we teach this?
+
+In addition to having a setup channel available upon login (with illustrated instructions), here are some links where community members can read more: 
+
+- [Discord Loves Open Source](https://discordapp.com/open-source)
+- [Discord Community Guidelines](https://discordapp.com/guidelines)
+- [How to use Discord](http://www.businessinsider.com/how-to-use-discord-the-messaging-app-for-gamers-2018-5)
+
+## Drawbacks
+
+### Supporting Learning vs Supporting Development
+
+There is some concern that there is already some confusion on Slack about where to get help learning/using Ember, and where to coordinate working on Ember. We need to have a clear delineation so that the folks who are spending their volunteer time to ship Ember features can continue to concentrate and do that.
+
+### Losing Community Members 
+
+There is some concern that we may lose some community members due to this move.
+
+## Alternatives
+
+The alternative to this would be to temporarily remain on Slack until we are able to evaluate and choose another viable option. However, we believe that staying on Slack is not desirable.
+
+## Unresolved questions
+
+* When will there be conversation threads?
```

Comment:
So I did some research on this, and they've improved search to the point where I think the addition of linking to a message will be a future enhancement, but I no longer consider it a blocker to adoption.

Reply:
Before threading the Slack channels used comment linking a lot more than currently (it's still used a good amount each day). SO I worry about context switching without links

Reply:
This may not be an issue much longer, some users are reporting being able to do this (I can't however) so it seems like this may be being rolled out using some feature flagging at the moment.

Reply:
I will add that the Discord team has been very responsive to feature requests from OSS communities in the past, and if anyone from the Ember community wants an invite to their private dev server to submit feature requests, let me know. I've been idling there since like 2015, when we mounted a campaign to support `code notation`